src/neuroconv/__init__.py

```diff
@@ -1,4 +1,3 @@
 from .nwbconverter import NWBConverter
 from .datainterfaces import *
-from .tools import spikeinterface, roiextractors, neo
```

Comment:
Similar to #76, I would not remove access to this alias from the top level. Breaks backcompatability and the decision to have this kind of structure stems from https://github.com/catalystneuro/nwb-conversion-tools/pull/411 and extensive in-person discussions around it. But we now know of ways to have access to that import structure without actually loading each respective module.